frontend/src/metabase/dashboard/components/AddSeriesModal.jsx

```diff
@@ -212,7 +212,7 @@ export default class AddSeriesModal extends Component {
                     <div className="flex-full ml2 mr1 relative">
                         <Visualization
                             className="spread"
-                            series={series}
+                            rawSeries={series}
```

Comment:
What's the rationale for renaming this `rawSeries`?

Reply:
See these new selectors:

https://github.com/metabase/metabase/blob/3e39686537be7c3ae2104078fbc67f52381a2d52/frontend/src/metabase/query_builder/selectors.js#L173-L200

Basically I wanted to have a distinction between an untransformed and a transformed series object. Maybe `rawSeries` isn't the best name for it, but I think it's good that now also in `Visualization` there isn't duplicate meaning for `series`.